docs/api/scheduler.md

```diff
@@ -1361,6 +1361,60 @@ Node details include host and rack name, capacity, resources, utilization, and a
 }
 ```
 
+## Node utilization
+
+Shows how node are distributed with regarding the utilization
```

Comment:
There are grammatical errors, maybe change it to this:
`Show how every node is distributed with regard to utilization.`